addons/mail/static/src/core/web/recipients_popover.js

```diff
@@ -0,0 +1,42 @@
+import { useService } from "@web/core/utils/hooks";
+
+import { Component, onWillStart } from "@odoo/owl";
+
+/**
+ * Override of the avatar card popover which shows an avatar card for specific users. This override applies
+ * the same idea but using `res.partner` instead of `res.users`. This enables this popover to be used
+ * to show a card with details for the recipients' partner. The card can also redirect the user to the
+ * res.partner form view if he wants more details or edit said partner.
+ */
+export class RecipientsPopover extends Component {
+    static template = "mail.RecipientsPopover";
+    static props = {
+        id: { type: Number, required: true },
+        close: { type: Function, required: true },
+        viewProfileBtnOverride: { type: Function },
+    };
+
+    setup() {
+        this.orm = useService("orm");
+        onWillStart(async () => {
+            [this.partner] = await this.orm.read("res.partner", [this.props.id], this.fieldNames);
+        });
+    }
+
+    get phone() {
+        return this.partner.phone;
+    }
+
+    get email() {
+        return this.partner.email_normalized || this.partner.email;
+    }
+
+    get fieldNames() {
+        return ["name", "email_normalized", "email", "phone"];
```

Comment:
do we really need the `email` field except for incomplete tests?
(not very important to change this)